core/variant/variant_utility.cpp

```diff
@@ -451,13 +451,15 @@ Variant VariantUtilityFunctions::lerp(const Variant &from, const Variant &to, do
 		case Variant::VECTOR4:
 		case Variant::QUATERNION:
 		case Variant::BASIS:
+		case Variant::TRANSFORM2D:
+		case Variant::TRANSFORM3D:
 		case Variant::COLOR:
 			break;
 		default:
 			r_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
 			r_error.argument = 0;
 			r_error.expected = Variant::NIL;
-			return R"(Argument "from" must be "int", "float", "Vector2", "Vector3", "Vector4", "Quaternion", "Basis, or "Color".)";
+			return R"(Argument "from" must be "int", "float", "Vector2", "Vector3", "Vector4", "Quaternion", "Basis", "Transform2D", "Transform3D" or "Color".)";
```

Comment:
```suggestion
			return R"(Argument "from" must be "int", "float", "Vector2", "Vector3", "Vector4", "Quaternion", "Basis", "Transform2D", "Transform3D", or "Color".)";
```
Keep the Oxford comma

Reply:
See [here](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything), it's obligatory in the engine and docs